app/controllers/deploys_controller.rb

```diff
@@ -2,6 +2,10 @@
 require 'csv'
 
 class DeploysController < ApplicationController
+  around_action :wrap_in_with_deleted, if: proc {
+    request.get? && (params[:with_deleted] == "true" || params.dig(:search, :deleted_at) == "true")
```

Comment:
can we use a single param ... otherwise passing it on with get complicated ...

Reply:
Right, but we have 2 different functionalities here.
1. list all deploys along with deleted deploys.
2. Filter and show only deleted deploys along with search results.

Reply:
Converted into reusable methods